### PR TITLE
Time picker new feature

### PR DESCRIPTION
**Pull request by Sina-pz**
This component's design has not finished yet

### CODE REVIEW

File: src/app/reusable/nv-time-picker/nv-time-picker.component.scss [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+    .spacer {
+      height: 7px;
+    }
+
+    .arrow-down {
+      width: 8px;
+      height: 5px;
+      color: $color-dark-gray-4d;
+
```
Comment (veeranjaneyuludavuluri) [inline src/app/reusable/nv-time-picker/nv-time-picker.component.scss:35]: Delete extra line

File: src/app/reusable/nv-time-picker/nv-time-picker.component.scss [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  ::ng-deep .ngx-timepicker-control__arrow {
+    font-size: 8px !important;
+    line-height: $line-height-medium !important;
+    color: $color-dark-gray-4d !important;
+    width: 8px !important;
+    height: 12px !important;
+    color: transparent !important;
+    z-index: 1;
+  }
+
```
Comment (veeranjaneyuludavuluri) [inline src/app/reusable/nv-time-picker/nv-time-picker.component.scss:99]: same

File: src/app/main/components/demo-page/demo-page.component.html [hunk 3]
```diff
@@ -76,6 +76,9 @@
       <div class="menu-item h40" (click)="onItemClick(nvComponentType.NvButton)">
         {{nvComponentType[nvComponentType.NvButton]}}
       </div>
+      <div class="menu-item h40" (click)="onItemClick(nvComponentType.NvTimePicker)">
```
Comment (veeranjaneyuludavuluri) [inline src/app/main/components/demo-page/demo-page.component.html:79]: If possible change h40 and others to more descriptive name in base.scss

**Review by veeranjaneyuludavuluri — APPROVED** [review]
Overall its looks good code wise. Resolve the comments I kept.

**Review by dhaval2989 — APPROVED** [review]
Good Job.

File: src/app/main/components/demo-page/demo-page.component.html (hunk 3 above)
Comment (Sina-pz) [reply src/app/main/components/demo-page/demo-page.component.html:79]: inspired by w3school but if you want you can update it to something better

**Pull request merged**